exercise2/src/Model/CalculatorModel.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace src\Model;
+
+class CalculatorModel
+{
+    
+    public $num1;
+    public $num2;
+    public $operator;
+
+    public function __construct ($num1, $num2, $operator) 
+    {
+        $this->num1 = $num1;
+        $this->num2 = $num2;
+        $this->operator = $operator;
+    }
+
+    public function compute(): int 
+    {
+        switch ($this->operator) 
+        {
```

Comment:
NIT. It's better if you inline opening curly braces for this kind of statements.